### PR TITLE
Code review request

### PR DESCRIPTION
**Pull request by abdellani**

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -2,15 +2,27 @@ import React from 'react';
...
   );
 };
 Button.propTypes = {
   name: PropTypes.string.isRequired,
+  color: PropTypes.bool,
+  width: PropTypes.bool.isRequired,
+};
+
+Button.defaultProps = {
+  color: false,
```
Comment (SunnyAsar) [inline src/components/Button.js]: Hi, lets write out code in a way that this default will be 'orange' string, that way we don't need to set undefined, the default will kick in.

**Review by SunnyAsar — CHANGES_REQUESTED** [review]
# Hi @abdellani 
well done on the project so far! it works!
- You have an interesting approach to the colour and wide props of the project. while the solution you have works, let's also try to stick to the bound of our project requirements and implement as such.
- with the code you have, colour props is always sent, but let's send it only when the colour is grey and have a default for the rest.

**Please make this update and request another review** Cheers.

**Comment by abdellani** [comment]
Hi @SunnyAsar  
Thank you for the review :)
The color is not always sent to button, because when I set color to `undefined` the component will use the default values.

**Comment by SunnyAsar** [comment]
but then colour is undefined and you have to do some logic on the button. what i mean is when the props is not needed like buttons that are orange, the colour should not be on it at all then the button can use the default.

**Comment by abdellani** [comment]
@SunnyAsar 
if I set color ==undefined, the default props.color will be used. it's exactly as if I didn't set color at all. 
I created a new pull request here https://github.com/abdellani/react-calculator/pull/7

**Comment by SunnyAsar** [comment]
@abdellani  Thanks for linking the new repo.

**Pull request merged**